examples/chat/iosApp/Configuration/Config.xcconfig

```diff
@@ -1,3 +1,3 @@
-TEAM_ID=
+TEAM_ID=ZN59W6SJ27
```

Comment:
I think it is better to leave TEAM_ID empty

Reply:
You're of course entirely correct – somehow this one always ends up in my commits by accident. 😱